packages/zostso/src/doc/IIssueTsoCmdResponse.ts

```diff
@@ -0,0 +1,21 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+export interface IIssueTsoCmdResponse {
+
+    cmdResponse: {message: string}[];
+
+    tsoPromptReceived: "Y" | "N",
+
+    servletKey?: string;
+
+    keywordDetected?: "Y" | "N",
+}
```

Comment:
Not requesting changes here, just a suggestion: It may be helpful from an extender's perspective to provide typedoc for the members in this interface.